python_scripts/linear_models_sol_03.py

```diff
@@ -8,17 +8,23 @@
 # %% [markdown]
 # # 📃 Solution for Exercise M4.03
 #
-# In all previous notebooks, we only used a single feature in `data`. But we
-# have already shown that we could add new features to make the model more
-# expressive by deriving new features, based on the original feature.
+# In the previous notebook, we showed that we can add new features based on the
+# original feature to make the model more expressive, for instance x**2 or x**3.
```

Comment:
The power expressions are misrendered:

<img width="261" alt="image" src="https://github.com/INRIA/scikit-learn-mooc/assets/89061/d2993fde-aee0-4e90-83a8-e946f0fb26f6">


```suggestion
# original feature to make the model more expressive, for instance `x ** 2` or `x ** 3`.
```